File: app/src/main/java/pl/pola_app/internal/di/OttoModule.java
package pl.pola_app.internal.di;

import com.squareup.otto.Bus;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class OttoModule {

    @Provides
    @Singleton
    Bus provideBus() {
        return new Bus();
    }
}

File: app/src/main/java/pl/pola_app/internal/di/OttoModule.kt
package pl.pola_app.internal.di

import com.squareup.otto.Bus
import dagger.Module
import dagger.Provides
import javax.inject.Singleton

@Module
class OttoModule {
    @Provides
    @Singleton
    fun provideBus(): Bus {
        return Bus()
    }
}